spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/trace/StackdriverTraceAutoConfiguration.java

```diff
@@ -112,25 +96,24 @@ public SleuthProperties stackdriverSleuthProperties(SleuthProperties sleuthPrope
 
 	@Bean
 	@ConditionalOnMissingBean
-	public Reporter<zipkin2.Span> reporter(
-			FlushableTraceConsumer traceConsumer,
-			SpanTranslator spanTranslator) {
-		return new StackdriverTraceReporter(
-				this.finalProjectIdProvider.getProjectId(),
-				traceConsumer,
-				spanTranslator);
+	public Reporter<Span> reporter(Sender sender, BytesEncoder<Span> encoder) {
```

Comment:
probably should just leave this one out, right? Otherwise it will ignore all sleuth's property overrides for the reporter, right?

Reply:
Interesting. I tried to leave it out and the spans weren't being sent.
Maybe I messed up, or maybe I missed any (auto-configuration) dependency?